src/Middleware/Zed/Process/ProcessConfig.php

```diff
@@ -3,10 +3,75 @@
 namespace Middleware\Zed\Process;
 
 use Middleware\Shared\Process\ProcessConstants;
+use Middleware\Zed\Process\Communication\Plugin\Hook\DummyPostProcessorHookPlugin;
+use Middleware\Zed\Process\Communication\Plugin\Hook\DummyPreProcessorHookPlugin;
+use Middleware\Zed\Process\Communication\Plugin\MapGeneratorMapperStagePlugin;
+use Middleware\Zed\Process\Communication\Plugin\MapGeneratorTranslatorStagePlugin;
+use Middleware\Zed\Process\Communication\Plugin\ProductImportMapperStagePlugin;
+use Middleware\Zed\Process\Communication\Plugin\ProductImportTranslatorStagePlugin;
+use SprykerMiddleware\Zed\Process\Communication\Plugin\Iterator\NullIteratorPlugin;
+use SprykerMiddleware\Zed\Process\Communication\Plugin\JsonReaderStagePlugin;
+use SprykerMiddleware\Zed\Process\Communication\Plugin\JsonWriterStagePlugin;
 use SprykerMiddleware\Zed\Process\ProcessConfig as SprykerMiddlewareProcessConfig;
 
 class ProcessConfig extends SprykerMiddlewareProcessConfig
 {
+    /**
+     * @return array
+     */
+    public function getProcessesConfig()
+    {
+        return [
+            ProcessConstants::MAP_GENERATOR_PROCESS => [
+                JsonReaderStagePlugin::PLUGIN_NAME,
+                MapGeneratorMapperStagePlugin::PLUGIN_NAME,
+                MapGeneratorTranslatorStagePlugin::PLUGIN_NAME,
+                JsonWriterStagePlugin::PLUGIN_NAME,
+            ],
+            ProcessConstants::PRODUCT_IMPORT_PROCESS => [
+                JsonReaderStagePlugin::PLUGIN_NAME,
+                ProductImportMapperStagePlugin::PLUGIN_NAME,
+                ProductImportTranslatorStagePlugin::PLUGIN_NAME,
+                JsonWriterStagePlugin::PLUGIN_NAME,
+            ],
+        ];
+    }
+
+    /**
+     * @return array
+     */
+    public function getProcessIteratorsConfig()
+    {
+        return [
+            ProcessConstants::PRODUCT_IMPORT_PROCESS => NullIteratorPlugin::PLUGIN_NAME,
+            ProcessConstants::MAP_GENERATOR_PROCESS => NullIteratorPlugin::PLUGIN_NAME,
+        ];
+    }
+
+    /**
+     * @return array
+     */
+    public function getPreProcessorHooksConfig()
+    {
+        return [
+            ProcessConstants::PRODUCT_IMPORT_PROCESS => [
```

Comment:
I guess it is still the case, that processes, that are not mentioned in every section will fail!